opensearch-project/OpenSearch-Dashboards
# [Workspace]Display duplicate collaborators error in add collaborators modal

### Description

This PR is for displaying duplicate collaborators errors in add collaborators modal after the "Add collaborators" button clicked. The collaborator input row will display a duplicate error like screenshot below if collaborator id duplicate with existing collaborators.

## Screenshot


<img width="879" alt="image" src="https://github.com/user-attachments/assets/0b8f3a16-3172-44be-8acf-5a9e84d1a8d7">


## Testing the changes



- Clone branch code and run `yarn osd bootstrap --single-version loose`
- Add below configs in `config/opensearch_dashboards.yml`
```
savedObjects.permission.enabled: true
workspace.enabled: true
uiSettings:
  overrides:
    'home:useNewHomePage': true
opensearchDashboards.dashboardAdmin.users: ['admin']
```
- Run `yarn start --no-base-path`
- Login with admin user
- Visit an existing workspace setting page
- Click "Add collaborators" button
- Click "Add Users" button
- Click "Add another User" and input two same collaborator id
- Click "Add collaborators" button, it should show duplicate collaborators error
- Remove duplicate collaborator id, and then click "Add collaborators" button, new collaborators will be saved
- Click "Add Users" button and input an existing collaborator id
- Click "Add collaborators" button, it should show duplicate collaborators error

## Changelog

- feat: [Workspace]Display duplicate collaborators error in add collaborators modal

### Check List
